checks/migrations/0016_domaintestappsecpriv_securitytxt_found_url.py

```diff
@@ -0,0 +1,27 @@
+# AddField Generated by Django 3.2.24 on 2024-03-09 12:49
+# Manually created SQL migration to handle old reports
+
+from django.db import migrations  # , models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("checks", "0015_add_rpki_scoring"),
+    ]
+
+    operations = [
+        # migrations.AddField(
+        #     model_name='domaintestappsecpriv',
+        #     name='securitytxt_found_url',
+        #     field=models.CharField(max_length=8000, null=True),
+        # ),
+        migrations.RunSQL(
+            sql=[
+                "ALTER TABLE checks_domaintestappsecpriv ADD COLUMN securitytxt_found_url VARCHAR(8000);",
```

Comment:
Maybe we can keep AddField and only use custom artisanal SQL for the UPDATE?

Reply:
It would probably also fix the build failure, as the migrations framework now does not know this SQL adds that field.